### DISCUSSION TITLE
Combine morpho distributions and aspect ratio

### DISCUSSION THREAD

**Discussion by asmaaharfoush** [neperfepx/neper#183] [Q&A] [answered]
Hello, I am a little bit confused about how we can combine any morpho distribution (lognormal) and aspect ratio, is the program override one (for instance the distribution) to do the aspect ratio, or how the program performs?

Code used: neper -T -n from_morpho -morpho 'diameq:lognormal(0.1,0.03),1-circularity:lognormal(0.145,0.03),aspratio(1,25)' -o 5 -id 5 -dim 2

Thanks!

**Comment by rquey** [comment, marked as answer]
Here is what the documentation specifies:
> When provided, other properties, such as the equivalent diameter or the sphericity (circularity, in 2D), are considered to apply to the cells as if they had no aspect ratio.

What the programs does is simply to generate the tessellation with its isotropic properties (no aspect ratio) and then to stretch it to get the specified aspect ratio. Before that, it applies the inverse stretching to the domain so that you get what you want it the end (both in terms of domain and cell aspect ratio).

**Reply by asmaaharfoush** [reply]
That is make sense, thank you!
